src/backend/events/filters/builtin/raid-viewer-count.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     name: "Raid Viewer Count",
     description: "Filter by how many viewers have been brought over by the raid.",
```

Comment:
The description needs a smidge of polish to reflect the new bidirectionality of the raid in/out events where it's used.
![filter text](https://github.com/user-attachments/assets/9be00509-ea5d-4cb2-8b04-e604c898c1b9)

Reply:
As far as actual verbiage, "Filter by how many users have been redistributed amongst the glorious interwebs by this raid." just seems like a poor choice, but is about the best I can come up with. I can only really point out that "brought over" in this context implies to me an incoming viewer count, while the filter is actually working upon the outgoing viewer count in this instance. Someone hire a copy-writer, stat!